unittests/snapshot_tests.cpp

```diff
@@ -459,17 +481,26 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_compatible_versions, SNAPSHOT_SUITE, snapshot
    //    now the test should pass.
    // 5. add the 3 new snapshot files in git.
    // -------------------------------------------------------------------------------------------------------------
-   if (save_snapshot)
-   {
-      // create a latest snapshot
-      auto latest_writer = SNAPSHOT_SUITE::get_writer();
-      base_chain.control->write_snapshot(latest_writer);
-      auto latest = SNAPSHOT_SUITE::finalize(latest_writer);
-
-      SNAPSHOT_SUITE::write_to_file("snap_" + current_version, latest);
+   // Save one copy of snapshots under Savanna only
```

Comment:
Took me a minute to figure out why you only want this under Savanna. The comment might be better as something like: `// Only want to save one snapshot, use Savanna as that is the latest`

Reply:
Thanks for your suggestion! I was struggling too about how to make it clear, trying to place the `only` in different places...